### ISSUE TITLE
cursor.execute should be able to handle rows as well as lists and tuples.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maximemaher/pyodbc#53]
```
What is the expected output? What do you see instead?
When trying to pass a single row to the execute statement, it complains and
says that it will only take a list or tuple.  It should be able to handle a
row since rows are iterateable.


What version of the product are you using? On what operating system?
2.1.6 on Cent OS 5



```

Original issue reported on code.google.com by `dkleeham...@gmail.com` on 4 Jun 2009 at 6:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `mkleehammer` on 6 Jun 2009 at 12:01
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I was working *up* the bug list instead of down and fixed Issue 57 which is 
actually 
a duplicate of this.  I hope you don't mind if I mark this as a duplicate since 
I've 
already checked in the fix and pushed it to git hub.

Will be fixed in 2.1.7
```

Original comment by `mkleehammer` on 8 Aug 2009 at 3:50
- Changed state: **Duplicate**
